parquet-column/src/main/java/org/apache/parquet/schema/LogicalTypeAnnotation.java

```diff
@@ -861,6 +871,36 @@ PrimitiveStringifier valueStringifier(PrimitiveType primitiveType) {
     }
   }
 
+  public static class UUIDLogicalTypeAnnotation extends LogicalTypeAnnotation {
```

Comment:
Why there is no implementation for hasCode() and equal()?

Reply:
As this is a singleton the default implementation of `equals(Object)` and `hashCode()` fits perfectly.

Reply:
Sounds good!